src/callbacks/rpm.h

```diff
@@ -397,12 +397,25 @@ struct InstallResolvableReportReceiver : public callback::ReceiveReport<target::
 ///////////////////////////////////////////////////////////////////
 struct FindFileConflictstReportReceiver : public callback::ReceiveReport<target::FindFileConflictstReport>
 {
+  static std::string mkProgressBarLabel( unsigned skipped_r = 0 )
+  {
+    // translators: A progressbar label
+    std::string ret { _("Checking for file conflicts:") };
+    if ( skipped_r ) {
+      // translators: progressbar label extension; %1% is the number of skipped items
+      static str::Format fmt { MSG_WARNINGString(" (%1% skipped)" ).str() };
+      ret += fmt % skipped_r;
+    }
+    return ret;
+  }
+
   virtual void reportbegin()
   {
+    Zypper::instance().out().gap();
+    _lastskipp = 0;
```

Comment:
should be named _lastskip ? I do not really mind about that, the pp just catched my eye